### DISCUSSION TITLE
Group type change and user-managed group owner change

### DISCUSSION THREAD

**Discussion by huwgeric** [watermint/toolbox#710] [Q&A]
Does Watermint support user-managed group owner change and group type change?
I didn't find the command options.
Thank you

**Comment by watermint** [comment]
Hello @huwgeric,
Sorry for delayed response. Unfortunately, there is no command to change group type now.
I'll add your request into my backlog.

**Reply by watermint** [reply]
Hi. @huwgeric,
You can now update type through the new command `group update type`
https://toolbox.watermint.org/commands/group-update-type.html

Please use Release 119 or later.
